src/app/static/src/js/components/runWorkflow/runWorkflowCreate.vue

```diff
@@ -114,8 +114,15 @@
                 }
             },
             rerun: function () {
-                if (this.selectedWorkflow) {
-                    const {reports, git_branch, git_commit, instances, name} = this.runWorkflowMetadata
+                if (this.selectedWorkflow && this.runWorkflowMetadata) {
+                    const {
```

Comment:
This whole clause can be simplified I think - should just be able to do 
```
  const runnableWorkflow = workflowRunDetailsToMetadata(this.runWorkflowMetadata);
  this.$emit("rerun", runnableWorkflow);
```
..as runWorkflowProgress does